supriya/scsynth.py

```diff
@@ -45,11 +49,14 @@ class Options:
     output_device: Optional[str] = None
     output_stream_mask: str = ""
     password: Optional[str] = None
+    port: int = DEFAULT_PORT
     protocol: str = "udp"
     random_number_generator_count: int = 64
     remote_control_volume: bool = False
+    realtime: bool = True
     restricted_path: Optional[str] = None
     sample_rate: Optional[int] = None
+    scsynth_path: Optional[str] = None  # deprecated, use `executable`
```

Comment:
Remove, since we're going with just `executable`

Reply:
Removed